he_twitter_miner/src/serverStartup.py

```diff
@@ -0,0 +1,15 @@
+from twitterMiner import TwitterMiner
+import logging
+import time
+
+if __name__ == '__main__':
+
+    while True:
+        try:
+            twitter_miner = TwitterMiner()
+            twitter_miner.start_mining()
+            
+        except Exception as e:
+            logging.error("exception received {}".format(e))
+        logging.info("waiting for 10 seconds before retrying")
```

Comment:
We should at least have a linear backoff here, with a retry limit.
Instead of waiting 10 seconds flat, do 30 seconds * # of failures (30, 60, 90). Also keep track of the # of failures, and if it's more than 5, break out of the `while` loop.